src/types.ts

```diff
@@ -70,6 +73,25 @@ export enum VaultStatusV2 {
   Zombie,
 }
 
+/**
+ * TODO: once the old enum VaultStatusV2 is removed, can rename to VaultStatusV2
+ */
+export enum VaultStatusV2_ {
```

Comment:
Imo the underscore make this name confusing, maybe we can name as `VaultStatusV2.1` or something?

Reply:
ok. will rename to the date of the upgrade `VaultStatusV2_202403`